tabcmd/commands/auth/session.py

```diff
@@ -211,22 +229,33 @@ def _read_existing_state(self):
 
     def _print_server_info(self):
         self.logger.info("=====   Server: {}".format(self.server_url))
+        if self.proxy:
+            self.logger.info("=====   Proxy: {}".format(self.proxy))
         if self.username:
             self.logger.info("=====   Username: {}".format(self.username))
+        if self.certificate:
+            self.logger.info("=====   Certificate: {}".format(self.certificate))
```

Comment:
Any reason why a user might not want the value of `self.certificate` in the logs?

Reply:
No, it's just a boolean. It probably needs a slightly better explanation in the docs: it says whether or not to use the local cert store to validate the server being connected to. Defaults to yes, use --no-certcheck to skip the check.